### PR TITLE
remove the redundant code in util.go

### PR DESCRIPTION
**Pull request by xichengliudui**
Remove the redundant code in util.go

### CODE REVIEW

File: pkg/util/utils.go [hunk 1]
```diff
@@ -34,11 +34,13 @@ import (
 	"k8s.io/minikube/pkg/minikube/out"
 )
 
-// ErrPrefix notes an error
-const ErrPrefix = "! "
+const (
```
Comment (josedonizetti) [inline pkg/util/utils.go]: Thanks for the PR. Do you mind also adding the `downloadURL` on the `const` block below to the same block here. pls?

Comment (xichengliudui) [reply pkg/util/utils.go]: @josedonizetti Thank you for your comments! 
Yes, I considered putting `downloadURL` into a `constant block` before I created this pr，but I don't think the two `constant blocks` define the same thing. all I didn't define together.
Are you sure? I can update pr

Comment (medyagh) [reply pkg/util/utils.go]: yes I agree with @josedonizetti it looks nicer to have all of them in same block.

**Comment by xichengliudui** [comment]
Thanks for your comments, updated!

**Comment by tstromberg** [comment]
@minikube-bot OK to test

**Comment by xichengliudui** [comment]
Is this due to my problem?
```
error: error pruning namespaced object apps/v1, Kind=StatefulSet: Get https://localhost:8443/apis/apps/v1/namespaces/kubernetes-dashboard/statefulsets?labelSelector=kubernetes.io%2Fcluster-service%21%3Dtrue%2Caddonmanager.kubernetes.io%2Fmode%3DReconcile: dial tcp [::1]:8443: connect: connection refused
```
```
 I0927 23:41:06.869526   15596 kubeadm.go:477] apiserver status: Error, err: <nil>
        I0927 23:41:07.377284   15596 kubeadm.go:161] https://10.138.0.2:8443/healthz response: <nil> &{Status:500 Internal Server Error StatusCode:500 Proto:HTTP/1.1 ProtoMajor:1 ProtoMinor:1 Header:map[Content-Length:[857] Content-Type:[text/plain; charset=utf-8] Date:[Fri, 27 Sep 2019 23:41:07 GMT] X-Content-Type-Options:[nosniff]] Body:0xc0005c18c0 ContentLength:857 TransferEncoding:[] Close:false Uncompressed:false Trailer:map[] Request:0xc0005ec800 TLS:0xc0007f0160}
        I0927 23:41:07.377375   15596 kubeadm.go:477] apiserver status: Error, err: <nil>
        I0927 23:41:07.876616   15596 kubeadm.go:161] https://10.138.0.2:8443/healthz response: <nil> &{Status:500 Internal Server Error StatusCode:500 Proto:HTTP/1.1 ProtoMajor:1 ProtoMinor:1 Header:map[Content-Length:[815] Content-Type:[text/plain; charset=utf-8] Date:[Fri, 27 Sep 2019 23:41:07 GMT] X-Content-Type-Options:[nosniff]] Body:0xc000632480 ContentLength:815 TransferEncoding:[] Close:false Uncompressed:false Trailer:map[] Request:0xc0005ec900 TLS:0xc0000e6370}
        I0927 23:41:07.876686   15596 kubeadm.go:477] apiserver status: Error, err: <nil>
        I0927 23:41:08.375661   15596 kubeadm.go:161] https://10.138.0.2:8443/healthz response: <nil> &{Status:200 OK StatusCode:200 Proto:HTTP/1.1 ProtoMajor:1 ProtoMinor:1 Header:map[Content-Length:[2] Content-Type:[text/plain; charset=utf-8] Date:[Fri, 27 Sep 2019 23:41:08 GMT]] Body:0xc000632600 ContentLength:2 TransferEncoding:[] Close:false Uncompressed:false Trailer:map[] Request:0xc0004dca00 TLS:0xc000345ef0}
```
there are so many errors!

**Comment by sharifelgamal** [comment]
@minikube-bot OK to test

**Comment by sharifelgamal** [comment]
@minikube-bot OK to test

**Comment by tstromberg** [comment]
Thank you!

**Pull request merged**